### PR TITLE
Update design of landing page

### LINKED ISSUE #18: Update landing grid with screenshots
And update aspect ratio of images

### PR DESCRIPTION
**Pull request by bonniezhou**
Fixes #18.

### CODE REVIEW

File: src/styles/App.scss [hunk 1]
```diff
@@ -14,37 +14,31 @@ body {
...
-
-.catalog-image-list-item {
-  text-decoration: none;
-  background-color: white;
-}
-
-.catalog-image-list-supporting {
-  padding-left: 20px;
+.catalog-image-list-item-container {
+  background-color: #f5f5f5;
```
Comment (moog16) [inline src/styles/App.scss]: These hardcoded values will probably used somewhere else. Want to make like a colors.scss file with these values? Or do you have any other ideas?

File: src/styles/App.scss [hunk 2]
```diff
@@ -14,37 +14,31 @@ body {
...
 }
 
 .catalog-image-list-label {
   font-weight: 500;
+  text-decoration: none;
+  color: #212121;
```
Comment (moog16) [inline src/styles/App.scss]: This one too

File: src/styles/App.scss (hunk 1 above)
Comment (bonniezhou) [reply src/styles/App.scss]: Updated it to use the Material color palette.

Comment (moog16) [reply src/styles/App.scss]: Nice! Looks good

**Pull request merged**